donut/modules/newsgroups/templates/group.html

```diff
@@ -2,82 +2,118 @@
-        <th>Email</th>
-        <th>Subscription</th>
-      </tr>
-      {% for applicant in applications %}
-        <tr>
-          <td> {{applicant.name}}</td>
-          <td> {{applicant.email}} </td>
-          <td> <a href={{ url_for('campus_positions')}}>Add to Group</a> </td>
-          <td> <a href={{ url_for('newsgroups.delete_application', user_id=applicant.user_id, group_id=group['group_id']) }}>Dismiss</a> </td>
-        </tr>
-       {% endfor %}
-     </table>
-  {% endif %}
-  </ul>
-  {% if actions['send'] or group['anyone_can_send'] %}
-    <br><a href={{ url_for('newsgroups.post', group_id=group['group_id']) }}>Send Message</a>
-  {% endif %}
-    <h2> Group Owners </h2>
-  {% for owner in owners %}
-    {{ owner.pos_name }}: <a href={{ url_for('directory_search.view_user', user_id=owner.user_id) }}>{{ owner.full_name }} </a><br>
-  {% endfor %}
-  {% if member %}
-    <h4> Recent Messages </h4>
-    <table style="width:100%">
-      <tr>
-        <th>Date</th>
-        <th>From</th>
-        <th>Subject</th>
-       </tr>
-       {% for message in messages %}
-         <tr>
-           <td> {{ message.time_sent }}</td>
-           <td> <a href={{ url_for('directory_search.view_user', user_id=message.user_id) }}>{{ message.post_as }}</a> </td>
-           <td> <a href={{ url_for('newsgroups.view_post', post_id=message.newsgroup_post_id) }}>{{ message.subject }}</a> </td>
-          </tr>
-        {% endfor %}
-    </table>
-    <a href={{ url_for('newsgroups.all_posts', group_id=group['group_id']) }}>View All</a>
-  {% else %}
-    <h4> Must be a member to view messages </h4>
-  {% endif %}
-{% endif %}
-</div>
+		{% if not group['visible'] and not member %}
+		<h2> Must be a member to view </h2>
+		{% else %}
+		<h2> {{ group['group_name'] }}
+			{% if actions['send'] or group['anyone_can_send'] %}
+			<a href={{ url_for('newsgroups.post', group_id=group['group_id']) }}><span class="glyphicon glyphicon-send"></span></a>
+			{% endif %}
+		</h2>
+		{% if group['group_desc'] %}
+		<h3> {{ group['group_desc'] }} </h3>
+		{% endif %}
+		<ul style="list-style-type:disc;">
+			{% if group['anyone_can_send'] %}
+			<li> Anyone can post to this group. </li>
+			{% else %}
+			<li> Posting is restricted to authorized members. </li>
+			{% endif %}
+			{% if group['admin_control_members'] %}
+			<li> Administrators control the membership of this group. </li>
+			{% endif %}
+			{% if group['visible'] %}
+			<li> Anyone can learn that this group exists. </li>
+			{% endif %}
+			<li> You are {% if not actions['receive'] %} not {% endif %} subscribed. </li>
+			{% if actions["control"] %}
+			<li> You have admin control. </li>
+			{% endif %}
+			{% if not member %}
+			<li>
+				<form method="POST" id="apply" action="{{ url_for('newsgroups.apply_subscription', group_id=group['group_id']) }}">
+					<div class="form-group">
+						<a href="#" onclick="$(this).closest('form').submit();">Apply for Subscription</>
```

Comment:
Should be `</a>`

Reply:
Also, can use `<button type='submit'>` instead to avoid the `onclick` handler

Reply:
I did that because I wanted it to be text because the button doesn't fit the aesthetic lol.

Reply:
`class="btn btn-link"` makes a button render like a link

Reply:
still looks like a button. I could add css to make the button look like a link or just keep `<a href="#" onclick="this.parentNode.submit()"`